src/app/dataset-block/metadata-block/components/event-details/components/common/engine-property/engine-property.component.ts

```diff
@@ -12,7 +11,16 @@ import { BasePropertyComponent } from "../base-property/base-property.component"
 export class EnginePropertyComponent extends BasePropertyComponent {
     @Input() public data: string;
 
-    public get descriptionEngine(): EventPropertyLogo {
-        return DataHelpers.descriptionForEngine(this.data);
+    public get engineLogo(): string | undefined {
+        return DataHelpers.descriptionForEngine(this.data).url_logo;
+    }
+
+    public get engineDescription(): string {
+        const engineDesc = DataHelpers.descriptionForEngine(this.data);
+        if (engineDesc.label && engineDesc.label.toLowerCase() != engineDesc.name.toLowerCase()) {
+            return `${engineDesc.label} (${engineDesc.name})`;
```

Comment:
Idea here is to show full name, but also how exactly the name looks in metadata:
![image](https://github.com/kamu-data/kamu-web-ui/assets/204914/4f673e51-68c5-4048-bf80-045941dc1b04)